### ISSUE TITLE
"Verify it's you" message blocking uploads

### ISSUE THREAD

**Issue by dinkster** [7x11x13/songs-to-youtube#16]
![2021-09-03_055340](https://user-images.githubusercontent.com/20588339/131995077-542c0a88-6f31-438c-8465-47de53b308d7.png)
I've been getting this message on every upload I try. I've tried redownloading my cookies, logging out and logging in, creating new profiles, nothing. Is there currently a workaround for this issue?

**Comment by 7x11x13** [comment]
I suspect that YouTube blocks certain IP ranges from uploading when they detect automation; I've been able to upload from my house and my sister's house, but when I moved to college I got blocked. I set up a VPN at my house as a workaround and that seems to be working so far. Maybe it would be possible to avoid detection by switching to something like pyppeteer_stealth.

**Comment by dinkster** [comment]
That's probably it; I started using the tool back at home during the summer and I'm using it for the first time after moving back to university. I don't really have access to a vpn at my home wifi but I can try to see if vpn-ing back to my hometown or using my phone as a hotspot does anything to fix it temporarily. If that doesn't work I'll try switching to pyppeteer_stealth.

**Comment by dinkster** [comment]
Alright, update: I've tried vpn-ing to my hometown and connecting to my phone's hotspot and neither worked. Next thing I tried was unhiding the browser window and just manually going through that verification process, but when I do that it blocks me because it's detected the automated browser. 
![image](https://user-images.githubusercontent.com/20588339/132144784-3cdc55b9-bcd8-428b-9fdb-73af3e62e7a8.png)

Bizarrely, enabling 2FA like #13 seemed to fix it though, at least for now. I'm currently connected to my college's wifi and everything seems to be uploading fine, no errors or pop-ups or anything. If it breaks again I'll submit another issue but this will work for now 👍
